packages/runtime/test/lib/testUtils.ts

```diff
@@ -700,12 +702,12 @@ export async function waitForRecipientToReceiveNotification(
 
 /**
  * The owner of a RelationshipAttribute receives a Request of a
- * peer and forwards them the ThirdPartyRelationshipAttribute,
+ * peer and forwards them the RelationshipAttribute,
  * waiting for all communication and event processing to finish.
  *
- * Returns the sender's own shared ThirdPartyRelationshipAttribute.
+ * Returns the sender's own shared RelationshipAttribute.
  */
-export async function executeFullRequestAndShareThirdPartyRelationshipAttributeFlow(
+export async function executeFullRequestExistingAttributeFlow(
```

Comment:
I think with this change we lose information. For example would I expect that the function can be used for IdentityAttributes, too. Also, from the naming it is not completely clear to me that the Request is answered.